### ISSUE TITLE
Update `axe-core` to 3.x for testing

### ISSUE THREAD

**Issue by thisisdano** [uswds/uswds#3216]
We currently test with axe-core `2.6.1` — let's try updating to `3.x` and re-test.

**Comment by mejiaj** [comment]
Created branch: `feature/USWDS-3216`

Right now there are 2 failing tests involving axe [here](https://circleci.com/gh/uswds/uswds/4495).

The axe violation is `scrollable-region-focusable` on a table element on all sizes. There's CSS that sets margin that seems to be causing the body to be "scrollable".
